apps/condo/domains/resident/schema/DiscoverServiceConsumersService.js

```diff
@@ -34,45 +49,136 @@ const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceCons
+                    const address = get(billingAccount, ['property', 'address'], null)
+                    const addressKey = get(billingAccount, ['property', 'addressKey'], null)
+                    const unitType = get(billingAccount, 'unitType', null)
+                    const unitName = get(billingAccount, 'unitName', null)
+
+                    let goFurther = false
+
+                    if (organizationType === SERVICE_PROVIDER_TYPE) {
+                        goFurther = true
+                    } else {
+                        const isFeatureFlagEnabled = await featureToggleManager.isFeatureEnabled(
+                            context,
+                            ENABLE_DISCOVER_SERVICE_CONSUMERS,
+                            { organization: organizationId },
+                        )
+
+                        if (isFeatureFlagEnabled) {
+                            goFurther = true
+                        }
+                    }
+
+                    if (goFurther) {
+                        items.push({ organizationId, address, addressKey, unitType, unitName })
+                    }
+                }
+
+                // Filter by acquiring context
+                const organizationsIdsWithAcquiringContext = new Set() //acquiringContexts.map((c) => get(c, ['organization', 'id']))
+                await loadListByChunks({
+                    context,
+                    list: AcquiringIntegrationContext,
+                    chunkSize: 50,
+                    where: {
                         deletedAt: null,
-                    })
+                        organization: { id_in: items.map(({ organizationId }) => organizationId) },
+                        status: CONTEXT_FINISHED_STATUS,
+                    },
+                    chunkProcessor: (chunk) => {
+                        chunk.forEach((row) => {
+                            organizationsIdsWithAcquiringContext.add(get(row, ['organization', 'id']))
+                        })
+                        return []
+                    },
+                })
 
+                items = items.filter((item) => organizationsIdsWithAcquiringContext.has(item.organizationId))
 
-                const getPayload = (accountNumber, resident) => ({
-                    dv,
-                    sender,
-                    resident: { connect: { id: resident.id } },
-                    organization: { connect: { id: get(resident, ['organization', 'id']) } },
-                    accountNumber: accountNumber,
+                // Filter by property
+                const organizationsIdsWithProperties = new Set()
+                await loadListByChunks({
+                    context,
+                    list: Property,
+                    chunkSize: 50,
+                    where: {
+                        deletedAt: null,
+                        organization: { id_in: items.map((item) => item.organizationId) },
```

Comment:
Orgnazization and address payloads must be combined

Reply:
Done